### PR TITLE
Update Model Tests

### PR DESCRIPTION
**Pull request by ChrisHuynh333**
## What does this PR do and why?
This PR fills in gaps within the model tests to improve coverage.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
1. Ensure tests pass locally

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: test/models/namespace_bot_test.rb [hunk 1]
```diff
@@ -33,4 +33,30 @@ def setup
...
+
+  test 'remove_membership_from_namespace' do
+    namespace_bot = namespace_bots(:project1_bot0)
+    membership = namespace_bot.user.members.find_by(namespace: namespace_bot.namespace)
+
+    assert_not_nil membership
+    assert namespace_bot.membership
+    assert_equal membership, namespace_bot.membership
+
+    namespace_bot.remove_membership_from_namespace
```
Comment (deepsidhu85) [inline test/models/namespace_bot_test.rb:59]: Currently the method `remove_membership_from_namespace` in `namespace_bot.rb` isn't working correctly with the `membership&.destroy`. If you update that to `user.members.find_by(namespace:).destroy` the method works and the underlying membership is removed

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Just have one comment

File: test/models/namespace_bot_test.rb (hunk 1 above)
Comment (ChrisHuynh333) [reply test/models/namespace_bot_test.rb:59]: Thanks, fixed in 2ff15e6a8e95b7f8df2181cf248fc0bbd039d04b

**Review by deepsidhu85 — APPROVED** [review]
Looks good to me!

**Pull request merged**